app/services/recipe_generator_service.rb

```diff
@@ -41,8 +42,17 @@ def system_message
   end
 
   def prompt
+    preference_info = @preferences.map { |pref|
+      "Consider this preference: #{pref.description}. Restriction: #{pref.restriction}"
+    }.join("\n")
     <<~CONTENT
-      Prompt goes here
+      You are an assistant that generates recipes. Your task is to create a detailed recipe only using the ingredients provided.
+       The user has the following preferences:#{preference_info}
+      Your response must be in JSON format, like this:
```

Comment:
For better prompting it is recommended to give clear instructions, like give a persona frame and a list of steps.
Also, some words like only, include, must is good to be in capital letters. Check this [recommendations](https://dev.to/get_pieces/10-prompt-engineering-best-practices-23dk).

```suggestion
      Your response MUST ALWAYS be in JSON format, like this:
```